### DISCUSSION TITLE
ik_solver.sample_configs has batch size of 0

### DISCUSSION THREAD

**Discussion by aadhithya-fauna** [NVlabs/curobo#442] [Research Q&A]
I trying to run the simple IK example with a custom urdf and when I try to sample q using the above function I get the issue of ik_solver.sample_configs has batch size of 0

Also if I use a custom joint positions to calculate the FK and then perform IK on those solutions. The IK solutions spit out are nan values

**Comment by balakumar-s** [comment]
> I trying to run the simple IK example with a custom urdf and when I try to sample q using the above function I get the issue of ik_solver.sample_configs has batch size of 0

This could be because there are no valid joint configurations (due to self collisions or world collisions). 

Can you try the IK solver with no collision checking, example is first code block here:
https://curobo.org/get_started/2a_python_examples.html#inverse-kinematics
